symfony/framework-bundle/4.3/config/packages/framework.yaml

```diff
@@ -0,0 +1,23 @@
+framework:
+    secret: '%env(APP_SECRET)%'
+    #default_locale: en
+    #csrf_protection: true
+    #http_method_override: true
+
+    # Enables session support. Note that the session will ONLY be started if you read or write from it.
+    # Remove or comment this section to explicitly disable session support.
+    session:
+        handler_id: ~
+        cookie_secure: auto
+        cookie_samesite: lax
+
+    #esi: true
+    #fragments: true
+    php_errors:
+        log: true
+
+    validation:
```

Comment:
This must be commented by default as validation is not available on `symfony/skeleton`.

Reply:
Or even better why not move this to the `symfony/validator` recipe instead?